ui/src/pages/Apps/index.tsx

```diff
@@ -17,8 +17,23 @@ export default class Apps extends React.Component<IProps> {
         const {} = this.props
         return (
             <div className="landing-content" style={{ marginTop: 20 }}>
-            <AppsWebPart/>
-         </div>
+                <AppsWebPart/>
+                <div className="contribute-notes">
```

Comment:
Adding the `div` hre leads to a duplication of the part, because it is also included in the `AppsWebPart` component. I'd recommend removing it here and adding the `contribute-notes` class to the div in `AppsWebPart`.

Reply:
I removed the div from the `AppsWebPart` index file since I didn't think it should be part of the code that loads and displays the app data.